python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -423,7 +423,7 @@ def makeWriteButler(cls, args: SimpleNamespace, taskDefs: Optional[Iterable[Task
                     # collection from its chain collection first.
                     with butler.transaction():
                         butler.registry.setCollectionChain(self.output.name, chainDefinition, flatten=True)
-                        butler.pruneCollection(replaced, purge=True, unstore=True)
+                        butler.removeRuns([replaced], unstore=True)
```

Comment:
I'm confused.  Didn't I already do that in d712735 ?

Reply:
Hmm, did I forgot to pull it?

Reply:
My fault, indeed, was branching off of old "main".